packages/components/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+#### Fixed
+
+- `Modal` scrollable modal does not jump to top on changes in react
```

Comment:
I think this would also happen in other frameworks since the root cause is a new object instance triggering the web component's change detection which causes a new lifecycle.